akka-http-core/src/test/scala/akka/http/impl/engine/parsing/RequestParserSpec.scala

```diff
@@ -667,6 +654,51 @@ abstract class RequestParserSpec(mode: String, newLine: String) extends AnyFreeS
           |
           |""" should parseToError(422: StatusCode, ErrorInfo("TRACE requests must not have an entity"))
       }
+
+      "a request with an unsupported transfer encoding" in new Test {
```

Comment:
Is there already a test checking that `Transfer-Encoding` and `Content-Length` must not be mixed? In any case, it might make sense to explicitly also add a test with multiple `Transfer-Encoding` entries and also a `Content-Length` header (in case we later add back some support for other encodings, it should then still fail but maybe with a different error message).

Reply:
I'l see if there is one and add one if not.

Reply:
Added

Reply:
Great, thanks!